contrib/certbot/template.py

```diff
@@ -25,11 +42,72 @@
 maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
 license = "Apache-2.0 AND MIT"
 url = "https://github.com/certbot/certbot"
-source = (
-    f"https://github.com/certbot/certbot/archive/refs/tags/v{pkgver}.tar.gz"
-)
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
 sha256 = "7e277bb461cae4071e22641e076d9232ae00ffda05bdb02832cbc1f862afab2d"
 
 
+def post_build(self):
+    for plugin in _plugins:
+        self.do(
+            "python",
+            "-m",
+            "build",
+            "--wheel",
+            "--no-isolation",
+            f"../{plugin}",
+        )
+
+
 def post_install(self):
     self.install_license("LICENSE.txt")
+    from cbuild.util import python
+
+    for plugin in _plugins:
+        self.do(
+            "python",
+            "-m",
+            "installer",
+            "--destdir",
+            self.chroot_destdir,
+            f"../{plugin}/dist/{plugin.replace('-','_')}-{pkgver}-py3-none-any.whl",
+        )
+    python.precompile(self, "usr/lib")
+
+
+@subpackage("certbot-plugins")
+def _pluggie(self):
+    self.pkgdesc = "certbot plugins"
+    self.install_if = [f"{pkgname}={pkgver}-r{pkgrel}"]
+    self.depends = depends + _plugins
```

Comment:
this isn't strictly correct as _plugins should be the name=pkgver-rpkgrel form for each

Reply:
it's probably easier to put an install_if in the _genmod on [pkgname=pkgver-rpkgrel, certbot-plugins=pkgver-rpkgrel]

Reply:
i could probably omit the self.depends line then since it's inherited from the templates' depends

Reply:
each plugin should individually depend on pkgname=pkgver-rpkgrel

Reply:
(so in genmod)

Reply:
sure